index.html

```diff
@@ -8,40 +8,217 @@
   
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css" rel="stylesheet">
+  <link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
 </head>
 <body>
     <header>
-      <a href="./index.html" class="logo">My Logo</a>
-      <nav>
-        <i class="hamburguer fa fa-bars"></i>
-      </nav>
+      <div class="header">
+        <a href="./index.html" class="logo">My Logo</a>
+        <nav>
+          <i class="hamburguer fa fa-bars"></i>
+        </nav>
+      </div>
     </header>
     <main>
       <section class="header-banner">
         <h2 class="name">I'm Oscar<br>Glad to see<br>you!</h2>
-        <p class="introduction">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <p class="introduction">I'm a software developer! I can help you build a product,
+           feature or website Look through some of my work and experience! 
+           If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
 
         <a href="https://linkedin.com/in/OscarFM-dev" class="connect-text">LET'S CONNECT</a>
         <ul class="social-media-icons">
           <li>
-            <a href="https://linkedin.com/in/OscarFM-dev"><img src="./images/linkedin.png" alt="linkedin logo" class="linkedin-logo"></a>
+            <a href="https://linkedin.com/in/OscarFM-dev">
+              <img src="./images/header-section/linkedin.png" alt="linkedin logo" class="linkedin-logo">
+            </a>
           </li>
           <li>
-            <a href="https://github.com/OscarFM-dev"><img src="./images/github.png" alt="github logo"></a>
+            <a href="https://github.com/OscarFM-dev">
+              <img src="./images/header-section/github.svg" alt="github logo">
+            </a>
           </li>
           <li>
-            <a href="https://twitter.com/OscarFMdev"><img src="./images/twitter.png" alt="twitter logo"></a>
+            <a href="https://twitter.com/OscarFMdev">
+              <img src="./images/header-section/twitter.svg" alt="twitter logo">
+            </a>
           </li>
           <li>
-            <a href="https://www.instagram.com/oscarfmdev/"><img src="./images/peace.png" alt="peace logo"></a>
+            <a href="https://medium.com/@oscarfernandezm95">
+              <img src="./images/header-section/medium.svg" alt="medium logo">
+            </a>
+          </li>
+          <li>
+            <a href="https://www.instagram.com/oscarfmdev/">
+              <img src="./images/header-section/peace.svg" alt="peace logo">
+            </a>
           </li>
         </ul>
       </section>
+
+      <section class="works">
```

Comment:
[OPTIONAL] Your portfolio is looking great, excellent work! Just a piece of advice, there's something going on with the works section when the height of the viewport changes, it overlaps the previous section:

<img width="493" alt="Screen Shot 2022-08-19 at 10 44 41 PM" src="https://user-images.githubusercontent.com/83717285/185727799-dcf4f682-5e3a-4b82-b057-f7187c12ede7.png">

<img width="499" alt="Screen Shot 2022-08-19 at 10 44 52 PM" src="https://user-images.githubusercontent.com/83717285/185727815-95e905c8-f2c4-4943-9cc1-d3802d46c004.png">

I would recommend fixing that bug before jumping into the desktop version, right now is easier to solve it.